test/integration/release-action-integration.ts

```diff
@@ -1,46 +1,87 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { expect } from 'chai'
-import { before, describe, test, after } from 'mocha'
+import { before, describe, test } from 'mocha'
 import { Environment } from '../../lib/entities/environment'
-import { Release } from '../../lib/entities/release'
+import { Release, ReleasePayload } from '../../lib/entities/release'
+import { ReleaseAction } from '../../lib/entities/release-action'
 import { Space } from '../../lib/entities/space'
 import { TestDefaults } from '../defaults'
-import { getDefaultSpace } from '../helpers'
+import { createTestSpace, initClient, initPlainClient } from '../helpers'
 import { makeLink } from '../utils'
 
-describe('ReleaseAction Api', async function () {
+describe('ReleaseAction Api', async () => {
   let testSpace: Space
   let testEnvironment: Environment
   let testRelease: Release
+  let testRelease2: Release
+
+  let testReleaseAction: ReleaseAction
+  let testReleaseAction2: ReleaseAction
 
   before(async () => {
-    testSpace = await getDefaultSpace()
+    testSpace = (await createTestSpace(initClient(), 'Release Actions')) as unknown as Space
     testEnvironment = await testSpace.getEnvironment('master')
-    testRelease = await testEnvironment.createRelease({
+
+    const contentType = await testEnvironment.createContentTypeWithId('testContentType', {
+      name: 'testContentType',
+      fields: [
+        {
+          id: 'title',
+          name: 'Title',
+          localized: false,
+          required: false,
+          type: 'Symbol',
+        },
+      ],
+    })
+    await contentType.publish()
+    await testEnvironment.createEntryWithId(
+      'testContentType',
+      TestDefaults.entry.testEntryReleasesId,
+      {
+        fields: {
+          title: { 'en-US': 'Non-localized value' },
+        },
+      }
+    )
+
+    const releasePayload: ReleasePayload = {
       title: 'Release (test)',
       entities: {
         sys: { type: 'Array' },
         items: [makeLink('Entry', TestDefaults.entry.testEntryReleasesId)],
       },
-    })
-  })
+    }
+
+    ;[testRelease, testRelease2] = await Promise.all([
+      await testEnvironment.createRelease(releasePayload),
+      await testEnvironment.createRelease(releasePayload),
```

Comment:
afaik there is no need for awaits inside of the array

Reply:
oh good catch 👍🏽